docs/user/_static/process/HashProcess_results.json

```diff
@@ -0,0 +1,94 @@
+{
+      {
```

Comment:
Looks like you trimmed off the opening line, so it's invalid json now. Could you add back in:
```
    "features" : [
```